dev-packages/node-integration-tests/package.json

```diff
@@ -47,6 +47,7 @@
     "graphql": "^16.3.0",
     "http-terminator": "^3.2.0",
     "ioredis": "^5.4.1",
+    "knex": "^2.5.1",
```

Comment:
Breaking change in major version 3 drops support for node < 16. See [here](https://github.com/knex/knex/releases/tag/3.0.0).